### PR TITLE
Change APIs to features

### PR DESCRIPTION
**Pull request by plinss**
Change 'API' to 'feature' when the principle applies to more than APIs
Add CSS naming convention
Fix capitalization in h2 for consistency

***
<a href="https://pr-preview.s3.amazonaws.com/plinss/design-principles/pull/207.html" title="Last updated on May 29, 2020, 6:34 AM UTC (32d897e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/207/f2c2921...plinss:32d897e.html" title="Last updated on May 29, 2020, 6:34 AM UTC (32d897e)">Diff</a>

### CODE REVIEW

**Comment by plinss** [comment]
Sorry, was rebasing onto master and force pushed to my fork, which closed the original PR...

Added a commit to address @cynthia's comment about device ids and did further cleanup in that and the previous section.

File: index.bs [hunk 1]
```diff
@@ -266,12 +266,12 @@ and *should* be in the [=ascii code point|ASCII range=].
 
 <h4 id="naming-consultation" class="no-num no-toc">Consultation</h4>
 
-Please consult widely on names in your APIs.
+Please consult widely on names in your features.
```
Comment (atanassov) [inline index.bs:269]: This is about API naming and should be reverted to talk about API rather than features.

File: index.bs [hunk 2]
```diff
@@ -1830,14 +1837,14 @@ about the user's file system.
...
 </div>
 
-When designing such an API,
-it may be necessary to also expose
-the fact that there are devices are available to be picked.
-Such a feature exposes
+When designing such a feature,
+it may be necessary to also include an API which exposes
+the fact that there are devices that are available to be picked.
+Such an API exposes
```
Comment (atanassov) [inline index.bs:1843]: This is JavaScript specific and should be called out as such.

File: index.bs [hunk 3]
```diff
@@ -47,7 +47,7 @@ url: https://drafts.csswg.org/css-transitions-1/#transitions; type: dfn; text: C
     }
 </style>
 
-<h2 id="basic-principles">Principles behind design of Web APIs</h2>
+<h2 id="basic-principles">Principles Behind Design of Web Features</h2>
```
Comment (atanassov) [inline index.bs:50]: We should add a section defining what we mean by API and how it differentiates from a Feature.

**Pull request closed without merging**